assets/js/atomic/blocks/product-elements/average-rating/block.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { useProductDataContext } from '@woocommerce/shared-context';
+import { useStyleProps } from '@woocommerce/base-hooks';
+import { __ } from '@wordpress/i18n';
+
+type ProductAverageRatingProps = {
+	className?: string;
+	textAlign?: string;
+};
+
+export const Block = (
+	props: ProductAverageRatingProps
+): JSX.Element | undefined => {
```

Comment:
Nit: I believe we can remove the "undefined" from here since we are always returning a JSX element, which will also resolve some TypeScript errors.